README.txt

```diff
@@ -235,5 +235,8 @@ FUTURE WORK
 - Allow SoftwareSerial or HardwareSerial to be used interchangeably?
 - CRC_XMODEM needs debugging, trying to test using a basic sketch and lrzsz
   I need a way to spy on what lrzsz is sending/receiving
-- Raspberry Pi Pico (RP2040) port
```

Comment:
I still plan to do a port to the RP2040 that does not depend on arduino dependencies that have been made available for the platform however for this to be properly checked off it needs a folder in `extras/ports` with sufficent instructions for how to get/install everything along with examples that can be easily compiled and run without needing to dive into the full documentation